src/demo/ui.jsx

```diff
@@ -391,22 +391,39 @@ const wordSet = {
   long: {
     text: ['Choose a new word to teach A.I.'],
     choices: [
+  //    [
```

Comment:
I would just delete content we don't want.  We can also revert if we want to bring them back.

Reply:
Thanks @breville. I can do that. Just considering these as placeholders. I'll update.